ccmlib/node.py

```diff
@@ -812,6 +812,13 @@ def run_sstablemetadata(self, output_file=None, datafiles=None, keyspace=None, c
         if output_file == None:
             return results
 
+    def get_sstables(self, output_file=None, datafiles=None, keyspace=None, column_families=None):
+        cdir = self.get_install_dir()
+        sstablemetadata = common.join_bin(cdir, os.path.join('tools', 'bin'), 'sstablemetadata')
+        env = common.make_cassandra_env(cdir, self.get_path())
+        sstablefiles = self.__gather_sstables(datafiles=datafiles, keyspace=keyspace, columnfamilies=column_families)
+        print_(sstablefiles)
```

Comment:
This command should return the sstablefiles, then the run() method in node_cmds.py should print out the returned value. This makes it more usable from other clients.